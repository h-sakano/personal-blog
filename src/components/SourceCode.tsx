import { Card } from 'antd';
import React from 'react';

const SourceCode = () => {
  return (
    <Card className="w-full" title="本ブログのソースコード" bordered={false}>
      <a
        className="flex items-center text-coloredLink"
        href="https://github.com/h-sakano/personal-blog"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        h-sakano/personal-blog
      </a>
    </Card>
  );
};

export default SourceCode;
