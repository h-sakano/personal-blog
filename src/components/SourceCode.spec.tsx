import { render } from '@testing-library/react';
import * as React from 'react';
import SourceCode from './SourceCode';

describe('<SourceCode />', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<SourceCode />);

    expect(asFragment()).toMatchSnapshot();
  });
});
