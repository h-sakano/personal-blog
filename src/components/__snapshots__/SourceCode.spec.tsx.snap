// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SourceCode /> should render correctly 1`] = `
<DocumentFragment>
  <div
    class="ant-card w-full"
  >
    <div
      class="ant-card-head"
    >
      <div
        class="ant-card-head-wrapper"
      >
        <div
          class="ant-card-head-title"
        >
          本ブログのソースコード
        </div>
      </div>
    </div>
    <div
      class="ant-card-body"
    >
      <a
        class="flex items-center text-coloredLink"
        href="https://github.com/h-sakano/personal-blog"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        h-sakano/personal-blog
      </a>
    </div>
  </div>
</DocumentFragment>
`;
